### PR TITLE
feat: pool performance chart

### PR DESCRIPTION
**Pull request by jpangelle**
### Description

This pull request implements the pool performance chart to the pool detail page redesign.

#1889

### Approvals

- [x] Dev
- [x] Designer
- [x] Product

### Screenshots

image

### Impact

Pool detail page

### CODE REVIEW

File: centrifuge-app/.env-config/.env.development [hunk 1]
```diff
@@ -1,21 +1,21 @@
-REACT_APP_COLLATOR_WSS_URL=wss://fullnode.development.cntrfg.com
```
Comment (jpangelle) [inline centrifuge-app/.env-config/.env.development:1]: will revert before merging

File: centrifuge-app/.env-config/.env.development
Comment (jpangelle) [inline centrifuge-app/.env-config/.env.development:1]: will revert this file before merging

File: centrifuge-app/src/components/Portfolio/PortfolioValue.tsx [hunk 2]
```diff
@@ -104,18 +104,26 @@ export function PortfolioValue({ rangeValue, address }: { rangeValue: string; ad
   )
 }
 
-const getRangeNumber = (rangeValue: string) => {
+export const getRangeNumber = (rangeValue: string, poolAge?: number) => {
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/Portfolio/PortfolioValue.tsx:107]: I like that you're reusing `getRangeNumber`! Would it make sense to move it to the `Charts/` folder and create a `utils.tsx` file?

**Review by sophialittlejohn — APPROVED** [review]
Looks good 🔥 just one suggestion and a reminder to revert the env file before merging!

**Comment by TimHoub** [comment]
Looks good - Only thing I would like to see changed is the alignment of the filter to the right of the chart. (Like in screenshot attached, red line is the outer grid)
Screenshot 2024-02-21 at 18 24 40

**Pull request merged**